Added the right side of the schema

MehakKambo opened the pull request.

LucaOnline commented:
Have you tried running this, yet? I think these need to be above the existing ones, since the existing ones depend on the ones you're adding. Also, if you're doing it in the same file, rename the file to `000-CreateTables.sql`, since we're only going to have the one file.

MehakKambo commented:
Updated the order, renamed the file, and tested.

![FP](https://user-images.githubusercontent.com/69995894/168876934-d6658b41-4e7d-4e60-b25c-3af6808c3364.PNG)
